products/angular.md

```diff
@@ -9,11 +9,17 @@ releaseDateColumn: true
 sortReleasesBy: 'release'
 changelogTemplate: https://github.com/angular/angular/releases/tag/__LATEST__
 releases:
+  - releaseCycle: "13"
+    release: 2021-11-04
+    support: 2022-05-04
+    eol: 2023-11-12
+    latest: "13.0.3"
```

Comment:
On https://github.com/angular/angular/releases the latest is `13.0.2`. Where do you have `13.0.3` from?

Reply:
mistyped ;-)

changed it.